scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -42,7 +41,7 @@ def _trace(A):
         return A.trace()
 
 
-def traceest(A, m3, seed=None):
+def traceest(A, m3, rng):
```

Comment:
If we need to re-run CI anyway, can we also:
```suggestion
def _traceest(A, *, m3, rng):
```